intro.md

```diff
@@ -30,11 +44,57 @@ similar forums for other software in the tree sequence [development community](h
 such as for [msprime](https://github.com/tskit-dev/msprime/discussions) and
 [tsinfer](https://github.com/tskit-dev/tsinfer/discussions).
 
-:::{note}
-Tutorials are under constant development. Those that are still a work in progress and
-not yet ready for use are shown in _italics_ in the list of tutorials.
 
-We very much welcome help developing existing tutorials or writing new ones. Please open
-or contribute to a [GitHub issue](https://github.com/tskit-dev/tutorials/issues) if you
-would like to help out.
-:::
+(sec_intro_running)=
+
+## Running tutorial code
+
+It is not necessary to run code in the tutorials yourself: tutorials can be followed by
+simply reading each page. However, it is also possible to run the tutorial code
+on your own computer, which will allow you to experiment with the examples provided.
```

Comment:
Bit wordy for me, how about

```suggestion
It is possible to run the tutorial code
on your own computer, if you wish. This will allow you to experiment with the examples provided.
```

Reply:
Sure, sounds good to be. Committed.